src/modules/createGame.js

```diff
@@ -0,0 +1,33 @@
+const createGame = (e) => {
+  e.preventDefault();
+
+  const name = document.getElementById('name');
+  const score = document.getElementById('score');
```

Comment:
[Optional]
- [ ] It will be really cool if apps validate name and score fields to ensure users cannot submit blank entries